src/types.jl

```diff
@@ -163,11 +163,11 @@ function AnnotationsSignal(records::Vector{Vector{TimestampedAnnotationList}})
     # treated similarly, i.e. the `SignalHeader` is overly trusted).
     max_bytes_per_record = maximum(sum(write_tal(IOBuffer(), tal) for tal in record)
                                    for record in records)
-    return AnnotationsSignal(Int16(cld(max_bytes_per_record, 2)), records)
+    return AnnotationsSignal(Int32(cld(max_bytes_per_record, 2)), records)
 end
 
 function SignalHeader(signal::AnnotationsSignal)
-    return SignalHeader("EDF Annotations", "", "", -1, 1, -32768, 32767,
+    return SignalHeader("EDF Annotations", "", "", -1, 1, -32768, 32767, # typemin/typemax
```

Comment:
I'm not sure the `typemin/typemax` comment is actually that informative; it may even be more confusing since the field is now typed as `Int32` but those values are not `typemin(Int32)`/`typemax(Int32)`, they're for `Int16`, which will be otherwise irrelevant after this PR.

Reply:
this field is still Int16 I think (it's the digital min/max)

Reply:
If it's creating any confusion at all, I'll just remove it.

Reply:
Perhaps this is only helpful for my weird brain 😅

Reply:
Ha, I think my confusion only arose because of the context of the change (I assumed without checking that it was related to the field that's having its type widened)